### PR TITLE
Add mark-as-unread

### PR DESCRIPTION
**Pull request by Alkarex**
https://github.com/FreshRSS/FreshRSS/issues/1966

### CODE REVIEW

File: app/i18n/en/feedback.php
Comment (Frenzie) [inline app/i18n/en/feedback.php]: The selected articles have been marked as read?

File: app/i18n/nl/index.php
Comment (Frenzie) [inline app/i18n/nl/index.php:43]: Markeer selectie als ongelezen

**Review by Frenzie — APPROVED** [review]
I'll toss in a follow-up PR for the Dutch translation if you prefer. :-)

File: app/layout/nav_menu.phtml
Comment (Frenzie) [inline app/layout/nav_menu.phtml:81]: I always get annoyed by languages that don't allow this. ;-)

File: app/i18n/nl/index.php
Comment (Frenzie) [reply app/i18n/nl/index.php:43]: Alright, I'll wait until several of these are merged (or at the very least until tomorrow) before I do that. :-)

File: app/layout/nav_menu.phtml
Comment (Alkarex) [reply app/layout/nav_menu.phtml:81]: Like JSON!

**Comment by Alkarex** [comment]
I have disabled *mark-as-unread* when read articles are not shown.
This is to prevent users from marking-as-unread the whole reader/category/feed by mistake. It might be possible to find a better logic (the ones I have in mind has the drawback of always requiring an additional expensive SQL request at each view to find the boundaries, or require some not-so-nice JavaScript).

**Comment by Alkarex** [comment]
image

**Comment by Alkarex** [comment]
@Frenzie Ready for translation PR ;-)

File: app/Controllers/entryController.php
Comment (Alkarex) [inline app/Controllers/entryController.php:67]: Fixed in https://github.com/FreshRSS/FreshRSS/pull/4456

**Pull request merged**
